avocado-setup.py

```diff
@@ -189,6 +189,26 @@ def get_avocado_bin():
         return avocado_binary
 
 
+def pip_install():
+    """
+    install package using pip
+    """
+    logger.info("install packages via pip interface")
+
+    pip_cmd = 'pip%s' % sys.version_info[0]
+
+    if CONFIGFILE.has_section('pip-package'):
+        package = CONFIGFILE.get('pip-package', 'package').replace(',', ' ')
+        cmd = '%s install %s' % (pip_cmd, package)
```

Comment:
let's install packages one by one, so that failure cases can be handled better...

Reply:
yes make sense , will do the changes

Reply:
Even though Praveen has done this already, here are my comments:
since we have distro packages install all at once, shouldn't we have a uniform way ?

Reply:
@narasimhan-v yes but we need to capture which package installation failed that help user to debug more so @sathnaga idea looks ok to me

Reply:
@PraveenPenguin Agreed. Looks ok to me as well.
@sathnaga time to have distro packages to install as well ?

Reply:
@narasimhan-v yes that will good suggestion